apps/logs.go

```diff
@@ -84,7 +84,10 @@ func dumpLogs(logsURL string, n int, filter string) error {
 		return nil
 	}
 
-	_, err = stdio.Copy(os.Stdout, res.Body)
+	//_, err = stdio.Copy(os.Stdout, res.Body)
+	buf := new(bytes.Buffer)
```

Comment:
With this method, you're putting everything in one big buffer, imagine there are A LOT of logs. That's something you want to do line by line with a bytes.StringReader for instance.